test/jdk/javax/swing/JFileChooser/6798062/bug6798062.java

```diff
@@ -153,6 +173,7 @@ private void setEnabledState(boolean enabled) {
     }
 
     private static void fail(String msg) {
+        PassFailJFrame.forceFail();
         throw new RuntimeException(msg);
```

Comment:
```suggestion
        PassFailJFrame.forceFail(msg);
```
`forceFail` will throw the exception on the main thread.